src/util/time.h

```diff
@@ -48,7 +58,11 @@ inline double CountSecondsDouble(SecondsDouble t) { return t.count(); }
 
 /**
  * DEPRECATED
- * Use either GetTimeSeconds (not mockable) or GetTime<T> (mockable)
+ * Use either ClockType::now() or Now<TimePointType>() if a cast is needed.
+ * ClockType is
+ * - std::chrono::sytem_clock  for system time
```

Comment:
s/sytem/system/

Between system_clock, steady_clock, and NodeClock, which would you see used for the CNodeStats?

Reply:
I think `NodeClock` for everything except for really short waits, like `ThreadMessageHandler` (which is already `steady_clock`). `system_clock` only for implementing `NodeClock` and logging?

Reply:
thx, fixed typo